web/EXAMPLES.md

```diff
@@ -24,15 +24,20 @@ For each third party asset request on the site, take the hostname of a request,
         - If a rule match is found, then for the first matching rule:
 
             a. If the rule has `action: ignore`, do not block[^2]
+	    
+	      b. If there are rule `options`, do any of the domains and request types match this request?[^1]
 
-            b. If there are rule `exceptions`, do any of the domains and request types match this request?[^1]
+	      - No: Don't block[^2]
```

Comment:
Shouldn't this skip to the next rule, rather than not block? I can compensate for this behaviour a bit with rule ordering if there's a fundamental incompatibility e.g., with MV3.

When we previously discussed this (and based on how the extension handles this), not matching `options` should act as a "skip this rule", rather than a block.

Reply:
Let me see if iOS did that the same way. Both ways would, but I'm inclined to go with what we already implemented.

Reply:
Is there a chance that some future rule will still change the outcome? Are we talking about CNAME?

Reply:
You could have a more general rule further down the list that it would block on. It's kinda a weird case, but I don't think it's incorrect this way. You didn't match the option (don't block) so you continue to check the rule list to see if you would block on anything else.

Reply:
Updated this from "don't block" to continue to next rule.